src/main/scala/mesosphere/marathon/core/readiness/ReadinessCheckResult.scala

```diff
@@ -7,5 +7,4 @@ case class ReadinessCheckResult(name: String,
                                 ready: Boolean,
                                 httpResponse: Option[HttpResponse])
```

Comment:
And if we rename this, do we need the explicit Format? 
Can't we just use `Json.format[ReadnissCheckResult]`?